src/meta-srv/src/region/supervisor.rs

```diff
@@ -359,7 +359,11 @@ impl RegionSupervisor {
         match self.is_maintenance_mode_enabled().await {
             Ok(false) => {}
             Ok(true) => {
-                info!("Maintenance mode is enabled, skip failover");
+                warn!(
+                    "Maintenance mode is enabled, skip failover. Removing failure detectors for regions: {:?}",
```

Comment:
The logs will be better:

> Skipping failover since maintenance mode is enabled. Removing failure detectors for regions: {:?}

Reply:
resolved in [7f09c0c](https://github.com/GreptimeTeam/greptimedb/pull/6022/commits/7f09c0c589f45937ca55ba3d05e3a40ffdc118b3)